File: iniciante/javascript/1150.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let x = parseInt(lines.shift())
let z = parseInt(lines.shift())
let i = 0
let soma = 0

while(z <= x){
    z = parseInt(lines.shift())
}

do{
    i++
    soma += x
    x++
}while(soma < z)

console.log(i)

File: iniciante/python/1150.py
# -*- coding: utf-8 -*-

soma = 0
z = 0
count_numeros = 0
x = int(input())
while True:
    z = int(input())
    if z > x:
        break

for i in range(x, z):
    soma += i
    count_numeros += 1
    if soma > z:
        break
print(count_numeros)
